src/NuGet.Server.Core/Infrastructure/ServerPackageRepository.cs

```diff
@@ -108,6 +108,35 @@ public class ServerPackageRepository
 
         private string CacheFileName => _settingsProvider.GetStringSetting("cacheFileName", null);
 
+        private TimeSpan InitialCacheRebuildAfter
+        {
+            get
+            {
+                const int defaultValue = 15;
+                var value = _settingsProvider.GetIntSetting("initialCacheRebuildAfterSeconds", defaultValue);
+                if (value <= 0)
+                {
+                    value = defaultValue;
+                }
+
+                return TimeSpan.FromSeconds(value);
+            }
+        }
+        private TimeSpan CacheRebuildFrequency
```

Comment:
Add empty line to separate from the property above?

Reply:
Fixed.